### PR TITLE
docs: add notes about upgrading for validators in the release notes

### PR DESCRIPTION
**Pull request by cmwaters**
Provide some information to validators on signalling to upgrade

### CODE REVIEW

File: docs/release-notes/release-notes.md [hunk 1]
```diff
@@ -13,6 +13,31 @@ This guide provides notes for major version releases. These notes may be helpful
   sed -i 's/ttl-num-blocks = 5/ttl-num-blocks = 12/' ~/.celestia-app/config/config.toml
   ```
 
+- Upgrades now use the `x/signal` module to coordinate the network to an upgrade height.
+
+The following command can be used, if you are a validator in the active set, to signal to upgrade to v3
+
+```bash
+celestia-appd tx signal signal 3 <plus transaction flags>
```
Comment (rach-id) [inline docs/release-notes/release-notes.md:21]: [personal opinion][nit]
```suggestion
celestia-appd tx signal signal 3 <transaction flags>
```
the plus seems a bit redundant

File: docs/release-notes/release-notes.md [hunk 2]
```diff
@@ -13,6 +13,31 @@ This guide provides notes for major version releases. These notes may be helpful
   sed -i 's/ttl-num-blocks = 5/ttl-num-blocks = 12/' ~/.celestia-app/config/config.toml
   ```
 
+- Upgrades now use the `x/signal` module to coordinate the network to an upgrade height.
+
+The following command can be used, if you are a validator in the active set, to signal to upgrade to v3
```
Comment (rach-id) [inline docs/release-notes/release-notes.md:18]:
```suggestion
The following command can be used, if you are a validator in the active set, to signal to upgrade to v3:
```

File: docs/release-notes/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,31 @@ This guide provides notes for major version releases. These notes may be helpful
...
+
+```bash
+celestia-appd tx signal signal 3 <plus transaction flags>
+```
+
+You can track the tally of signalling by validators using the following query
```
Comment (rach-id) [inline docs/release-notes/release-notes.md:24]:
```suggestion
You can track the tally of signalling by validators using the following query:
```

File: docs/release-notes/release-notes.md [hunk 4]
```diff
@@ -13,6 +13,31 @@ This guide provides notes for major version releases. These notes may be helpful
...
+Once 5/6+ of the voting power have signalled, the upgrade will be ready. There is a hard coded delay between confirmation of the upgrade and execution to the new state machine. 
+
+To view the upcoming upgrade height use the following query:
+
+```bash
+celestia-appd query signal upgrade
+> An upgrade is pending to app version 3 at height 2348907.
+```
+
+For more information refer to the module [docs](../../x/signal/README.md)
```
Comment (rach-id) [inline docs/release-notes/release-notes.md:39]:
```suggestion
For more information refer to the module [docs](../../x/signal/README.md).
```

**Review by rach-id — APPROVED** [review]
some personal opinion nits

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**